.gitignore

```diff
@@ -15,8 +15,6 @@ coverage/
 .mailmap
 component.json
 debug/local/
-Gemfile.lock
```

Comment:
I've checked in the lockfile as is [recommended](https://bundler.io/guides/rationale.html#checking-your-code-into-version-control) by the Bundler docs. This prevents accidental upgrades of dependencies that might break the process.